.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: daRGjMGroJCvOxe8wY5y3VAjP2SyplCx3
```

Comment:
can't we use that as a secret?
similar to how it's being done in github actions
https://github.com/marketplace/actions/coveralls-github-action

also. might be a good idea to consider github actions as it's allinone place solution

Reply:
The consensus is to use CircleCI over GitHub Actions (I added both)